crates/frontend/src/pages/default_config.rs

```diff
@@ -26,19 +29,35 @@ pub struct RowData {
 #[component]
 pub fn default_config() -> impl IntoView {
     let tenant_rs = use_context::<ReadSignal<String>>().unwrap();
+    let enable_grouping =
+        create_rw_signal(get_local_storage::<bool>("enable_grouping").unwrap_or(false));
+    let (filters, set_filters) = create_signal(if enable_grouping.get() {
```

Comment:
This will cause hydration issues, as enable_grouping is only available in localstorage, and the list filters will update the number of list items.

Reply:
@mahatoankitkumar this needs to be done in this way only: https://github.com/juspay/superposition/pull/209/files#diff-f14ee882e4ca49f81aa2293a656f92603255460bc68c7247b510d60ad6f7c4e6R64

or we start accepting cookies for this

Reply:
Sure i leave it as it is, but cookies has the same problem, already tried it.